Docs/Getting Started with the GE Reflective Memory Component.md

```diff
@@ -10,61 +10,59 @@ Before starting this exercise, install the following software:
 - Data Sharing Framework ([DSF Releases](https://github.com/ni/niveristand-data-sharing-framework-custom-device/releases))
 - Data Sharing Framework Plugins ([DSF Plugins Releases](https://github.com/ni/niveristand-data-sharing-framework-custom-device-plugins/releases))
 - RT driver support for the GE Reflective Memory device
-   - **_Incompatibility_**: The GE Reflective Memory driver for this custom device is incompatible with the GE Fanuc driver installed through MAX, which is used by VeriStand's built-in reflective memory support. You must uninstall the GE Fanuc driver from the RT target before continuing.
-   - Locate the GE5565PIORC_NetworkInterrupts_DMA.inf file on disk. The .inf file is installed by the NI Package for this component and can be found on disk at the file path below:
+   **Note** The GE Reflective Memory driver for this custom device is incompatible with the GE Fanuc driver installed through NI-MAX. The GE Fanuc driver is used by VeriStand's built-in reflective memory support. You must uninstall the GE Fanuc driver from the RT target before continuing.
```

Comment:
@WillCawley , this note appears mid-sentence and formatted unexpectedly to me. Can you confirm this is what you want?

Reply:
Hi @niphilj. It is not really mid sentence. It is a note on an item. Would it be better if we rephrased the item to be more like this: "GE Reflective Memory RT driver support"?

Reply:
Hey @WillCawley , when looking at the document in the viewer (shown in the image), this is why it appears mid-sentence. You may have it on a separate line in the editor, but I don't see it that way.
![image](https://user-images.githubusercontent.com/60665206/92612517-9141d300-f27f-11ea-8621-007313124be1.png)

Reply:
Okay. I think I figured out the issue. MarkDown reacts poorly to bold text inside ordered lists. I'll look for some workarounds

Reply:
Thanks @WillCawley ! And like I mentioned, many of your additions that have special text formatting show this issue. Please look over the preview of the rendered markdown to make sure you get them all since I didn't tag them all.